packages/react-components/src/components/tabs/VerticalTab.tsx

```diff
@@ -0,0 +1,51 @@
+import React,  {useContext, useCallback, useState} from 'react';
+import {FormContext} from '@aemforms/af-react-renderer';
+import {getRenderer, useFormIntl} from '@aemforms/af-react-renderer';
+import {ContainerJson, State, CUSTOM_PROPS_KEY, TRANSLATION_ID, getOrElse} from '@aemforms/af-core';
+
+const VerticalTab = (props: State<ContainerJson>) => {
```

Comment:
Why are you not using useRuleEngine here? Have you tested the rules?
Please use the hook here same as TextFIeld.
don't localize the label here use the hook for this.

Reply:
Implemented all the changes!